exporters/otlp-http/trace/build.gradle.kts

```diff
@@ -0,0 +1,34 @@
+plugins {
+    id("otel.java-conventions")
+    // TODO: uncomment once ready to publish
+    // id("otel.publish-conventions")
+
+    id("otel.animalsniffer-conventions")
+    id("com.github.johnrengelman.shadow")
+}
+
+description = "OpenTelemetry Protocol HTTP Trace Exporter"
+otelJava.moduleName.set("io.opentelemetry.exporter.otlp.http.trace")
+
+dependencies {
+  api(project(":sdk:trace"))
+
+  implementation(project(":exporters:otlp:common"))
+
+  implementation("com.squareup.okhttp3:okhttp")
+  implementation("com.squareup.okhttp3:okhttp-tls")
+  implementation("com.squareup.okio:okio")
+
+  testImplementation(project(":sdk:testing"))
+
+  testImplementation("com.linecorp.armeria:armeria-junit5")
+}
+
+tasks {
+  shadowJar {
```

Comment:
This is going to result in publishing two artifacts, one with a `-all` classifier. I think if we shade we should publish the non-shaded version as well, so this seems ok. The `-shaded` classifier is probably easier to understand and used by some projects like gRPC.

That being said, I think I would like to go ahead and leave this out until someone asks for it - it's not hard to add a shaded version later, but in the meantime, documenting the two is a bit painful.